apps/judicial-system/scheduler/infra/judicial-system-scheduler.ts

```diff
@@ -0,0 +1,23 @@
+import { ref, service, ServiceBuilder } from '../../../../infra/src/dsl/dsl'
+
+export const serviceSetup = (services: {
+  backend: ServiceBuilder<'judicial-system-backend'>
+}): ServiceBuilder<'scheduler'> =>
+  service('scheduler')
```

Comment:
Please change the name to 'judicial-system-scheduler'

Reply:
@pshomov Can I remove the "image" declaration? We are not using it in any of the other applications.

Reply:
Name fixed